src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -742,7 +743,7 @@ protected function pagination( $which ) {
 		$current              = $this->get_pagenum();
 		$removable_query_args = wp_removable_query_args();
 
-		$current_url = set_url_scheme( 'http://' . esc_url_raw( $_SERVER['HTTP_HOST'] ) . esc_url_raw( $_SERVER['REQUEST_URI'] ) );
+		$current_url = set_url_scheme( esc_url_raw( $_SERVER['REQUEST_URI'] ), 'relative' );
```

Comment:
There was a comment here about usage of the variable.
We should address what the bot mentioned here, even tho the old code was already like that.

Reply:
This was not resolved, we need to make sure we look into this.